besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1188,9 +1248,32 @@ private JsonRpcConfiguration jsonRpcConfiguration() {
     jsonRpcConfiguration.setAuthenticationEnabled(isRpcHttpAuthenticationEnabled);
     jsonRpcConfiguration.setAuthenticationCredentialsFile(rpcHttpAuthenticationCredentialsFile());
     jsonRpcConfiguration.setAuthenticationPublicKeyFile(rpcHttpAuthenticationPublicKeyFile());
+    jsonRpcConfiguration.setTlsConfiguration(tlsConfiguration);
     return jsonRpcConfiguration;
   }
 
+  private TlsConfiguration getRpcHttpTlsConfiguration() {
+    final String password;
+    try {
+      password =
+          Files.asCharSource(rpcHttpTlsKeyStorePasswordFile.toFile(), Charsets.UTF_8)
+              .readFirstLine();
+      if (password == null) {
+        throw new ParameterException(
+            commandLine, "Unable to read keystore password for JSON-RPC HTTP endpoint");
+      }
+    } catch (final IOException e) {
+      throw new ParameterException(
+          commandLine, "Unable to read keystore password file for JSON-RPC HTTP endpoint", e);
+    }
+
+    return TlsConfiguration.TlsConfigurationBuilder.aTlsConfiguration()
```

Comment:
nit: Given you've done all the heavy lifting a builder is probably overkill. Maybe worth getting another opinion, but to me, the builder acted as a converter between cmdline options and a basic TlsConfiguration. If the conversion is being done here, the builder is possibly not required ... thoughts?